httpx/_client.py

```diff
@@ -1130,6 +1205,22 @@ async def request(
         allow_redirects: bool = True,
         timeout: typing.Union[TimeoutTypes, UnsetType] = UNSET,
     ) -> Response:
+        """
+        Build and send a request.
+
+        Roughly equivalent to:
```

Comment:
Should this just be "Equivalent to:"?

Reply:
Fair point, because currently *it is*. But do you want to have some leeway in case it stops being, for some reason?